clients/src/main/resources/common/message/LeaderChangeMessage.json

```diff
@@ -22,7 +22,9 @@
     {"name": "LeaderId", "type": "int32", "versions": "0+",
       "about": "The ID of the newly elected leader"},
     {"name": "Voters", "type": "[]Voter", "versions": "0+",
-      "about": "The voters who voted for the current leader"}
+      "about": "The voters who voted at the time of election"},
+    {"name": "EndorsingVoters", "type": "[]Voter", "versions": "0+",
```

Comment:
Could we use "Granting" instead of "Endorsing"? I think this is a more consistent term considering `LeaderState`.